src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -69,40 +69,39 @@ public static class CommandLine
                 //validArgs array as well present in the CommandSettings.cs
                 public static class Args
                 {
-                    public static readonly string Agent = "agent";
-                    public static readonly string Auth = "auth";
-                    public static readonly string CollectionName = "collectionname";
-                    public static readonly string DeploymentGroupName = "deploymentgroupname";
-                    public static readonly string DeploymentPoolName = "deploymentpoolname";
-                    public static readonly string DeploymentGroupTags = "deploymentgrouptags";
-                    public static readonly string EnvironmentName = "environmentname";
-                    public static readonly string EnvironmentVMResourceTags = "virtualmachineresourcetags";
-                    public static readonly string MachineGroupName = "machinegroupname";
-                    public static readonly string MachineGroupTags = "machinegrouptags";
-                    public static readonly string Matrix = "matrix";
-                    public static readonly string MonitorSocketAddress = "monitorsocketaddress";
-                    public static readonly string NotificationPipeName = "notificationpipename";
-                    public static readonly string NotificationSocketAddress = "notificationsocketaddress";
-                    public static readonly string Pool = "pool";
-                    public static readonly string ProjectName = "projectname";
-                    public static readonly string ProxyUrl = "proxyurl";
-                    public static readonly string ProxyUserName = "proxyusername";
-                    public static readonly string SslCACert = "sslcacert";
-                    public static readonly string SslClientCert = "sslclientcert";
-                    public static readonly string SslClientCertKey = "sslclientcertkey";
-                    public static readonly string SslClientCertArchive = "sslclientcertarchive";
-                    public static readonly string SslClientCertPassword = "sslclientcertpassword";
-                    public static readonly string StartupType = "startuptype";
-                    public static readonly string Url = "url";
-                    public static readonly string UserName = "username";
-                    public static readonly string WindowsLogonAccount = "windowslogonaccount";
-                    public static readonly string Work = "work";
+                    public const string Agent = "agent";
```

Comment:
Unfortunately since these are used in Attributes they need to be const.
@jpricket gave me a good but verbose alternative which would be to define the variable twice i.e.
```
public readonly string Agent = Agent_Internal;
internal const string Agent_Internal = "agent";
```

@jtpetty , @zachariahcox  do we think it's worth doing.  It would be verbose but safer if we think external libraries are using these vars.